app/urls.py

```diff
@@ -25,9 +25,8 @@
     url(r'^engage/$', views.engage, name='engage'),
 
     # social login urls
-    url('', include('social.apps.django_app.urls', namespace='social')),
+    url(r'^api/v1/', include('social_django.urls', namespace='social')),
```

Comment:
Eta te "^api/v1/" ta lagche? Why not leave it blank?

Reply:
hmm r'^api/v1/' r ki mane?